src/views/Alteration.vue

```diff
@@ -216,6 +216,8 @@ export default class Alteration extends Mixins(CommonMixin, FeeMixin, FilingTemp
 
   @Watch('hasBusinessNameChanged')
   @Watch('hasBusinessTypeChanged')
+  @Watch('getNameRequestLegalName')
+  @Watch('getEntityType')
   onBusinessTypeChanged () {
```

Comment:
This is to fix the fee summary bug - 
sometimes the EntityType and BusinessName did change, but hasBusinessNameChanged/hasBusinessTypeChanged is not changed (keeps being true)
so add these 2 watchers.